### PR TITLE
Added selection content-type

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | noe
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR introduces a selection content-type for Articles.

#### TODO

- [x] Tests (ArticleController cget with ids)
- [x] Type-Tabs in overlay

### CODE REVIEW

**Comment by alexander-schranz** [comment]
@wachterjohannes  It should be able to click on the whole row to select the article not just on the checkbox in the overlay.

File: Resources/public/dist/components/article-selection/contentItem.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<a href="#" class="link" data-id="<%= id %>">
```
Comment (alexander-schranz) [inline Resources/public/dist/components/article-selection/contentItem.html]: its not possible to open this link in a new tab as it is not replaced with the correct url:

also a normal click redirects to undefined locale: `/#articles/undefined/edit:9a81b285-3015-4f0f-9b3c-3aecdec0f5bf/details`

**Comment by wachterjohannes** [comment]
@alexander-schranz fixed

**Comment by alexander-schranz** [comment]
ACK

**Pull request merged**